index.html

```diff
@@ -0,0 +1,169 @@
+            </ul>
+            <ul class="nav_link">
+                <li><a class="nav_link_line" href="#">About</a></li>
+                <li><a class="nav_link_line" href="#">concert</a></li>
+                <li><a class="nav_link_line" href="#">ticket</a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="homepage_content">
+                <div class="header_content">
+                    <h3>"welcome  y'all"</h3>
+                    <h1 class="h1_image">Monsters of rock world concert 2023</h1>
+                </div>
+                <div class="paragraph">
+                    <p>A great concert celebrating multiple talents from various counties <br>  
+                        which is taking place in gotham city on 15th october</p>
+                </div>
+                <div class="paragraph_two">
+                    <h2>2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p>@lugogo cricket oval </p>
+                </div>
+            </div>
+        </section>
+        <section class="work_section">
+            <div class="work_section_content">
+                <div class="main_program">
+                    <h2>Main program</h2>
+                    <div class="line_program"></div>
+                </div>
+                <div class="program_details_wrapper">
+                    <div class="program_details">
+                        <img src="images/program_icon_01.png" alt="">
+                        <h3>Artist</h3>
+                        <p> Rock music, also called rock and roll, rock & roll, or rock 'n' roll </p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_02.png" alt="">
+                        <h3>Band</h3>
+                        <p>Big Rock Band, Hollywood, Florida. 3628 likes · 51 talking about this. </p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_04.png" alt="">
+                        <h3>Artist</h3>
+                        <p>Rock music, also called rock and roll, rock & roll, or rock 'n' roll</p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_05.png" alt="">
+                        <h3>band</h3>
+                        <p> Big Rock Band, Hollywood, Florida. 3628 likes · 51 talking about this. </p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_05.png" alt="">
+                        <h3>Artist</h3>
+                        <p> Rock music, also called rock and roll, rock & roll, or rock 'n' roll</p>
+                    </div>
+                </div>
+                <button class="ticket_button">get your concert ticket</button>
```

Comment:
- [ ] On the mobile version, this button is overlapping with the next section. I think it would be great if you fix it. 🤝️


| Your design                                                             | Template                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/217841290-46fffb89-7a25-4022-9f6d-72ff7320a4a1.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/217841227-c61499fa-701f-43e6-b2be-4f3eb578b616.png" width="300">|